src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.5
++++++
+* Add support for virtual network operations.
```

Comment:
It would be better to specify what the new commands or command groups area.
`* Added 'postgres/myql server vnet-rule' commands.`